File: lanjii/lanjii-admin-ui/src/types/sys/sysRole.ts
/** 系统角色 */
export interface SysRole {
  /** 角色ID */
  id?: number
  /** 角色名称 */
  name: string
  /** 角色编码 */
  code: string
  /** 显示顺序 */
  sortOrder?: number
  /** 是否启用（1启用 0禁用） */
  isEnabled?: number
  /** 备注 */
  remark?: string
  /** 创建时间 */
  createTime?: string
  /** 更新时间 */
  updateTime?: string
  /** 创建人 */
  createBy?: string
  /** 更新人 */
  updateBy?: string
  /** 删除标志（0-未删除，1-已删除） */
  deleted?: number
}

File: lanjii/lanjii-modules/module-system/system-biz/src/main/java/com/lanjii/sys/entity/SysRole.java
package com.lanjii.sys.entity;

import com.baomidou.mybatisplus.annotation.TableName;
import com.lanjii.sys.base.AdminBaseMapper;
import com.lanjii.system.api.dto.SysRoleDTO;
import com.lanjii.system.api.vo.SysRoleVO;
import com.lanjii.framework.mp.base.TenantBaseEntity;
import lombok.Data;
import lombok.EqualsAndHashCode;
import org.mapstruct.Mapper;
import org.mapstruct.factory.Mappers;

/**
 * 角色表(SysRole)表实体类
 *
 * @author lanjii
 */
@Data
@EqualsAndHashCode(callSuper = true)
@TableName("sys_role")
public class SysRole extends TenantBaseEntity {

    /**
     * 角色ID
     */
    private Long id;

    /**
     * 角色名称
     */
    private String name;

    /**
     * 角色编码
     */
    private String code;

    /**
     * 显示顺序
     */
    private Integer sortOrder;

    /**
     * 是否启用（1启用 0禁用）
     */
    private Integer isEnabled;

    /**
     * 备注
     */
    private String remark;

    @Mapper
    public interface SysRoleMapper extends AdminBaseMapper<SysRole, SysRoleVO, SysRoleDTO> {

    }

    public static final SysRoleMapper INSTANCE = Mappers.getMapper(SysRoleMapper.class);

}
